### PR TITLE
Kochan/Task_2

### PR DESCRIPTION
**Pull request by Nacosh**
completed task 2

### CODE REVIEW

File: Tasks/Kochan/Task_2.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+class People:
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Kochan/Task_2.py:1]: OK, so tell me, why do we need to have this class?

File: Tasks/Kochan/Task_2.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
+dict_raw = {9103976271: [("Reina", "Meinhard"), ("Memphis", "Tennessee")],
+            4199392609: [("Stephanie", "Bruce"), ("Greensboro", "North Carolina")],
+            9099459979: [("Ermes", "Angela"), ("Dallas", "Texas")],
+            6123479367: [("Lorenza", "Takuya"), ("Indianapolis", "Indiana")],
+            7548993768: [("Margarete", "Quintin"), ("Raleigh", "North Carolina")]}
+
+sprav1 = People()
+sprav1.add_dict(dict_raw)
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Kochan/Task_2.py:43]: why don't use some init for the data uploading?

**Review by MikitaTsiarentsyeu — APPROVED** [review]
I will add 20 points to your 100, but I disagree with the usage of the class from the logical point of view, we do not need such complex solution to do such simple task

**Comment by Nacosh** [comment]
Hi, NikitaThanks for feedbackRe init - just haven't crossed my mind )Class - we really don't need class there, but I needed it just to train since Stepik provided only tasks like Please create Class A and Calss B.. Too straightforward ) -- С уважением,Наталья Кочан 8 044 500 29 238 0296 53 33 10    23.08.2022, 19:26, "MikitaTsiarentsyeu" ***@***.******@***.*** approved this pull request.I will add 20 points to your 100, but I disagree with the usage of the class from the logical point of view, we do not need such complex solution to do such simple taskIn Tasks/Kochan/Task_2.py:> @@ -0,0 +1,44 @@
+class People:
OK, so tell me, why do we need to have this class?In Tasks/Kochan/Task_2.py:> +sprav1 = People()
+sprav1.add_dict(dict_raw)
why don't use some init for the data uploading?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Pull request merged**